.palantir/revapi.yml

```diff
@@ -0,0 +1,2 @@
+versionOverrides:
```

Comment:
For the upstream version, I agree it should be `0.13.0`, which is the tag you're using. But should this mark it as 0.13.1? What happens if we make the branch 0.13.0 instead?

Reply:
This is marking it as 0.13.0 because of the output of `git describe`. I believe if we pushed a tag for `release-base-0.13.1`, it would use that instead.

However, it seems we can use an override in the `build.gradle` file instead: https://github.com/palantir/gradle-revapi#configuration

Let me try that.

Reply:
Setting it to 0.13.0 then compares the output of current master (or the generated code from this branch) to the `0.13.0` artifact.

Reply:
I set it by running `./gradlew iceberg-api:revapiVersionOverride --replacement-version 0.13.0`

Reply:
If we tagged a `release-base-0.13.1`, we'd also be able to compare that from within the same file.

I left the full output comparison in a gist. https://gist.github.com/kbendick/b9543ac8498bb06815a8c4e993f5c969

Reply:
I think we want to compare against the last minor release right now. Later, we'll want to compare to the last major release.